conf/base.config

```diff
@@ -22,32 +22,24 @@ process {
 
   // Process-specific resource requirements
   // See https://www.nextflow.io/docs/latest/config.html#config-process-selectors
-  withLabel:process_very_low {
-    cpus = { check_max( 2 * task.attempt, 'cpus' ) }
-    memory = { check_max( 6.GB * task.attempt, 'memory' ) }
-    time = { check_max( 3.h * task.attempt, 'time' ) }
-  }
   withLabel:process_low {
-    cpus = { check_max( 4 * task.attempt, 'cpus' ) }
-    memory = { check_max( 10.GB * task.attempt, 'memory' ) }
+    cpus = { check_max( 2 * task.attempt, 'cpus' ) }
```

Comment:
Are those nf-core wide defaults or defined by us?

Reply:
I think we overwrote them

Reply:
we defo need single_threaded and very_low

Reply:
Readded both.

Can revert the changes for the other ones if you want me to.